rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -277,16 +300,18 @@ void Player::play()
 void Player::pause()
 {
   clock_->pause();
+  RCLCPP_INFO_STREAM(this->get_logger(), "Pausing play.");
```

Comment:
Minor stylistic nitpick (across diffs in this file), I prefer to omit `this` in C++. This codebase may not be fully consistent on it, but I think on average it's `this`-less

```suggestion
  RCLCPP_INFO_STREAM(get_logger(), "Pausing play.");
```